packages/react/src/input/Input.tsx

```diff
@@ -0,0 +1,46 @@
+import { type ComponentPropsWithRef, forwardRef } from "react";
+
+import { Box } from "../box";
+// import { Text } from "../text";
+import { type ExtendProps } from "../utils";
+import { type Recipe, recipe } from "./Input.recipe";
+
+type InputProps = ExtendProps<
+  ComponentPropsWithRef<"input">,
+  ComponentPropsWithRef<typeof Box>,
+  { isDisabled?: boolean; isInvalid?: boolean } & Recipe
+>;
+
+export const Input = forwardRef<HTMLInputElement, InputProps>(
+  (
+    {
+      isDisabled,
+      isInvalid,
+      placeholder,
+      size = "md",
+      type,
+      variant = "default",
+      ...props
+    },
+    ref,
+  ) => {
+    return (
+      <Box
+        aria-invalid={isInvalid}
+        asChild
+        data-disabled={isDisabled}
+        display="flex"
+        flexDirection="column"
+        fontFamily="sans"
+        fontSize="md"
+        maxW="sm"
+        {...recipe({ size, variant })}
+        {...props}
+      >
+        <input placeholder={placeholder} ref={ref} type={type} />
```

Comment:
let's spread the `...props` onto the `<input />` component instead - so we will pick certain props like `className` to be on the outer wrapper/box - but everything else will get forwarded to inner input element

Reply:
See https://github.com/optimizely-axiom/optiaxiom/pull/41#discussion_r1629601071